metrotranscom/doorway
# feat: change layout of ListingsCombined for mobile vs desktop

# Pull Request Template

## Issue Overview

This PR addresses #133 


## Description

- Separate CSS for ListingsCombined into its own file.
- Create different styles for desktop vs mobile. The desktop style is the existing style with 2 columns. The mobile style has the map in the background and the listings list in the foreground, covering only half the screen and with rounded corners.

Not included, will be in next PR:
- Expand/hide listings list on swipe.

## How Can This Be Tested/Reviewed?

Bring up the public site and go to the listings page.
Adjust the screen width/use the Chrome Device Emulator to see what the page looks like for desktop vs mobile.

Desktop:
![image](https://github.com/metrotranscom/doorway/assets/67125998/33c42481-d769-4b9c-8e3f-a0514bd2ca61)

Mobile:
![image](https://github.com/metrotranscom/doorway/assets/67125998/67df2cc9-aaf0-4ae1-8e25-098c368a5070)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
